test/helpers/ssh_command.go

```diff
@@ -163,7 +163,7 @@ func runCommand(session *ssh.Session, cmd *SSHCommand) (bool, error) {
 	}
 	outChan := copyWait(cmd.Stdout, stdout)
 
-	if err = session.Run(cmd.Path); err != nil {
+	if err = session.Start(cmd.Path); err != nil {
```

Comment:
I'm not sure that it's this simple, unfortunately, due to the fact that we are calling `copyWait` on the stdout and stderr here. I think that if we use `Run` instead of start, we may block forever below (lines 170 to 176).

Reply:
Instead, if we use `RunCommandContext`, I don't think we need to call `session.Close()` after calling `runCommand` at all, since I'm fairly sure (but not certain) the session is 'closed' after we 'run' it. I asked for clarification on this on an issue I filed upstream querying the semantics of SSH sessions in golang: https://github.com/golang/go/issues/32453#issuecomment-502871914 .